driver-core/src/test/java/com/datastax/driver/core/MaterializedViewMetadataTest.java

```diff
@@ -29,6 +30,13 @@
 @CassandraVersion(major = 3)
 public class MaterializedViewMetadataTest extends CCMBridge.PerClassSingleNodeCluster {
 
+    @Test(groups = "unit")
+    public void should_use_all_fields_in_equals_and_hashCode() {
+        EqualsVerifier.forClass(MaterializedViewMetadata.class)
+            .allFieldsShouldBeUsedExcept("cassandraVersion", "keyspace")
```

Comment:
cassandraVersion is likely not considered since we don't expect it to be different between MVs.  Compared within MVs in same keyspace so keyspace doesn't factor in?